source/tutorials/using-api-interface.rst

```diff
@@ -166,7 +45,7 @@ ${JWT}``, which contains the token got from auth component.
 
 
 Device creation
```

Comment:
The input/output boxes with lines are unintelligible, the line numbers do not match with their content. It really disturbs the reading and understanding process.

Reply:
It's a bug in the theme. I was trying to find a solution but they work only locally (as far as I could figure out, the files that needs to be changed can't be modified because of readthedocs things)

Reply:
Does this problem still happen to you? I checked the rendered version in my own readthedocs, and all line numbers are properly - ahm - aligned with the code.

Reply:
No anymore :)